grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -299,7 +299,7 @@ class CycleCountService {
         cycleCounts.each { CycleCountDto cycleCount ->
             cycleCount.cycleCountItems.each { CycleCountItemDto item ->
                 data << [
-                        "Cycle count id": cycleCount.id,
+                        "Product cycle count id": cycleCount.id,
```

Comment:
it was Manon's request to make this column "Product cycle count id" so that it is more understandable by the users.